docs/src/components/MetaData.astro

```diff
@@ -0,0 +1,35 @@
+---
+import { site } from '../config.ts';
+const { content = {}, canonicalURL } = Astro.props;
+let formattedContentTitle = content.title ? `${content.title} 🚀 ${site.title}` : site.title;
+---
+
+<!-- OpenGraph Tags -->
+<meta property="og:title" content={formattedContentTitle}/>
+<meta property="og:type" content="article"/>
+<meta property="og:url" content={canonicalURL}/>
+<meta property="og:locale" content={content.ogLocale ? content.ogLocale : site.ogLocale}/>
```

Comment:
do we have an example of using this today? I might say "YAGNI" and push for us to add these content frontmatter overrides when we need them (or if we ever want to open source this component for others to use).

Reply:
There's a similar pattern in the `<html {content.lang ?? 'en-US'}>`, which on second thought I'll at least update the syntax to that...

@Hanawa02 had expressed interest in making sure the docs were ready for i18n translation, and I figured it'd be a good idea to get it setup? :thinking:

Reply:
There's a similar pattern in the `<html {content.lang ?? 'en-US'}>`, which on second thought I'll at least update the syntax to that...

@Hanawa02 had expressed interest in making sure the docs were ready for i18n translation, and I figured it'd be a good idea to get it setup? :thinking:

Reply:
Yeah, at least I thought that at least for small items we could already set it up slowly.